lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -147,6 +158,7 @@ def main():
             db=dict(required=True),
             ext=dict(required=True, aliases=['name']),
             state=dict(default="present", choices=["absent", "present"]),
+            cascade=dict(type='bool', default='no'),
```

Comment:
Sorry didn't notice this first time around but I think should this be a boolean not a string.
```suggestion
            cascade=dict(type='bool', default=False),
```

Reply:
Indeed. Thank you